gax-grpc/src/main/java/com/google/api/gax/grpc/OperationCallable.java

```diff
@@ -27,148 +27,112 @@
+          ApiCallContextDecorator callContextDecorator,
+          OperationCallSettings settings) {
+    return new OperationCallable<>(callableImpl, callContextDecorator, settings);
+  }
 
-  /** Private for internal use. */
   private OperationCallable(
-      UnaryCallable<RequestT, Operation> initialCallable,
-      ClientContext clientContext,
-      RetryingExecutor<Operation> executor,
-      OperationsClient operationsClient,
-      ApiFunction<Operation, ResponseT> responseTransformer,
-      ApiFunction<Operation, MetadataT> metadataTransformer) {
-    this.initialCallable = checkNotNull(initialCallable);
-    this.clientContext = checkNotNull(clientContext);
-    this.executor = checkNotNull(executor);
-    this.operationsClient = checkNotNull(operationsClient);
-    this.responseTransformer = checkNotNull(responseTransformer);
-    this.metadataTransformer = checkNotNull(metadataTransformer);
+      OperationCallableImpl<RequestT, ResponseT, MetadataT, OperationT> callable,
+      ApiCallContextDecorator callContextDecorator,
+      OperationCallSettings settings) {
+    this.callable = Preconditions.checkNotNull(callable);
+    this.callContextDecorator = callContextDecorator;
+    this.settings = settings;
   }
 
   /**
-   * Creates a callable object that represents a long-running operation. Public only for technical
-   * reasons - for advanced usage
-   *
-   * @param settings {@link OperationCallSettings} to configure the method-level settings with.
-   * @param clientContext {@link ClientContext} to use to connect to the service.
-   * @param operationsClient {@link OperationsClient} to use to poll for updates on the Operation.
-   * @return {@link OperationCallable} callable object.
+   * Returns the {@link OperationCallSettings} that contains the configuration settings of this
+   * OperationCallable.
    */
-  public static <RequestT, ResponseT extends Message, MetadataT extends Message>
-      OperationCallable<RequestT, ResponseT, MetadataT> create(
-          OperationCallSettings<RequestT, ResponseT, MetadataT> settings,
-          ClientContext clientContext,
-          OperationsClient operationsClient) {
-    UnaryCallable<RequestT, Operation> initialCallable =
-        settings.getInitialCallSettings().create(clientContext);
-
-    RetryAlgorithm<Operation> pollingAlgorithm =
-        new RetryAlgorithm<>(new OperationResponsePollAlgorithm(), settings.getPollingAlgorithm());
-    ScheduledRetryingExecutor<Operation> scheduler =
-        new ScheduledRetryingExecutor<>(pollingAlgorithm, clientContext.getExecutor());
-
-    return new OperationCallable<>(
-        initialCallable,
-        clientContext,
-        scheduler,
-        operationsClient,
-        new ResponseTransformer<>(new AnyTransformer<>(settings.getResponseClass())),
-        new MetadataTransformer<>(new AnyTransformer<>(settings.getMetadataClass())));
+  public OperationCallSettings getSettings() {
+    return settings;
```

Comment:
I have already removed all of the `getSettings` methods in the XCallable classes in my local version - update forthcoming.